### ISSUE TITLE
Unable to change wallpaper

### ISSUE THREAD

**Issue by wout** [elementary/switchboard-plug-pantheon-shell#156]
I'm unable to choose another wallpaper. Even when I select a new one, close the switchboard and re-open it again, the newly chosen wallpaper is selected, but the old one is still showing.

![screenshot from 2018-11-18 08-51-20](https://user-images.githubusercontent.com/107324/48669973-c3fd6300-eb0f-11e8-8831-7900f850eabb.png)

Only after logging out, the wallpaper has been changed. Also, in multitasking view, the background either disappears or is chopped off to the resolution of the built-in display and stretched to fill the screen (as in stretched the last column of pixels to fill the empty space).

**Comment by cassidyjames** [comment]
Thanks for reporting! This is actually a duplicate of https://github.com/elementary/gala/issues/13

**Comment by wout** [comment]
Ok, couldn't find it at first sight.

**Comment by stucash** [comment]
I am experiencing the same thing but might be different behaviour behind the scene. #158
